crates/typst-library/src/model/quote.rs

```diff
@@ -101,11 +101,11 @@ pub struct QuoteElem {
     ///
     /// #show quote.where(block: false): it => {
     ///   ["] + h(0pt, weak: true) + it.body + h(0pt, weak: true) + ["]
-    ///   if it.attribution != none [ (#it.attribution)]
+    ///   if it.attribution != none [ (#it.attribution) ]
```

Comment:
Just some thoughts for discussion: I wonder if the example should be updated to better reflect this change? E.g. add some text around it so the space is shown. But not sure what would be the best phrase to go along with it.